<!-- 目录跳转(整个header部分)-->
<!-- 跳转去的页面是href后面的文件名-->
<!-- 跳转js还没写，我之后会补上 -->
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" type="text/css" href="./style.css">
    <style>
        .tt p{
            /* text-align: justify; */
            line-height: normal;
        }
    </style>
</head>
<body class="blue">
    <header>
        <div class="headbar"></div>
        <div class="navbar">
            <div class="titlebox">
                <img class="logo" src="./logo.png" alt="">
            </div>
            <div class="titlebox">
                <a href="./home.html">
                    Home
                </a>
            </div>
            <div class="titlebox">
                <span><img src="https://cdn.builder.io/api/v1/image/assets/TEMP/50553b738128039c507a6801435406eb1469bc028992b71cf38e93f00f12b4a6?" alt="" class="btn-gra"></span>
                <a href="./introduction.html">
                    Intro
                </a>
                <!-- <div class="dropdown-content">
                    <a href="{{ url_for('pages', page = 'team') }}">Members</a>
                    <a href="{{ url_for('pages', page = 'attribution') }}">Attributions</a>
                    <a href="{{ url_for('pages', page = 'collaboration') }}">Collaboration</a>
                  </div> -->
            </div>
            
            <div class="titlebox">
                <a href="./search.html">
                    Search
                </a>
            </div>
            <div class="titlebox">
                <a href="./statistics.html">
                    Statistics
                </a>
            </div>
            <!-- <div class="titlebox">
                <a href="">
                    Inter-Chrom
                </a>
            </div> -->
            <div class="titlebox">
                 <span><img src="https://cdn.builder.io/api/v1/image/assets/TEMP/50553b738128039c507a6801435406eb1469bc028992b71cf38e93f00f12b4a6?" alt="" class="btn-gra"></span>
                <a href="./tutorial.html">
                    Tutorial
                </a>
            </div>
            <div class="titlebox">
                <a href="./download.html">
                    Download
                </a>
            </div>
            <div class="titlebox" style="border-right: 0;">
                <a href="./about.html">
                    About
                </a>
            </div>
        </div>
    </header>
    <!-- 侧边栏 -->
    <div id="sidebar1" style="height: 16.7w ;top: 17vw;">
        <a class="sidebar-link" data-percentage-from="2" data-percentage-to="8" href="#" onclick="scrollToPosition(6); return false;">Search</a>
        <a class="sidebar-link" data-percentage-from="2" data-percentage-to="8" href="#" onclick="scrollToPosition(73); return false;">Download</a>
        <a class="sidebar-link" data-percentage-from="2" data-percentage-to="8" href="#" onclick="scrollToPosition(85); return false;">File Format</a>
        <a class="sidebar-link" style="border-bottom: none;" data-percentage-from="2" data-percentage-to="8" href="#" onclick="scrollToPosition(120); return false;">Visualization</a>
    </div>


    <div class="boxbk white">
        <h1 class="darkblue" style="font-size: 2vw;">Tutorial</h1>
        <br>
        <div class="tt">
            <h3 class="darkblue">Search</h3>
            <hr class="darkblue">
            <p>Here, we demonstrate how to obtain samples that meet the criteria using our search and filter functions.</p>
            <img src="./tt1.png" alt="">
            <p style="margin-top: 1vw; margin-bottom: 1vw;">◆Users can enter a GSM or GSE number in the input box and click the "Search" button. The retrieved data will be displayed in the table below. Clicking on the GSE or GSM number in the table will redirect to the corresponding GEO page for the sample. Clicking on peaks.bed, loops.bed, or annotation.txt will download the respective files. Clicking on the sample ID will navigate to the sample's detail page.</p>
            <img src="./tt3.png" alt="">
            <p style="margin-top: 1vw;margin-bottom: 1vw;">◆Users can select the data that meets their requirements in the filter box. We have divided the filter criteria into five categories: Methods, Cell Type, Cell Line, TF, and Health states. Each filter box allows for multiple selections. After making selections, users need to click the "Done" button below. If no selection is made in a filter box, all options are selected by default. At any point during the selection process, users can click "Clear" to reset all options to their default state. After completing the selections in all five filter boxes, users can click the "Search" button. Data that meets the criteria will be displayed in the table below. The distribution of the samples meeting the criteria across these five categories will be displayed in a pie chart.</p>
            <img style="width: 32vw; margin: auto;" src="./tt4.png" alt="">
            <img src="./tt2.png" alt="">
        </div>
        <div style="text-align: justify;" class="tt">
            <h3 class="darkblue">Download</h3>
            <hr class="darkblue">
            <p>To access and download the data files for HiChIP and ChIA-PET, please visit the Download page. There, you will find links to compressed archives for each data type.</p>
            <!-- <p>2.xxxxxxxxxxxxxxxxxxxxxxxxxxxx</p> -->
            <p>HiChIP: Peak, Loop, and Annotation data</p>
            <img src="./tt5.png" alt="">
            <p>ChIA-PET: Peak, Loop, and Annotation data</p>
        </div>
        <div style="text-align: left !important;" class="tt">
            <h3 class="darkblue">File Format</h3>
            <hr class="darkblue">
            <h4 class="darkblue">HiChIP peak File</h4>
            <p>◆ <b>chrom:</b> Chromosome name</p>
            <p>◆ <b>chromStart:</b> Start position of the feature </p>
            <p>◆ <b>chromEnd:</b> End position of the feature</p>
            <p>◆ <b>name:</b> Feature name</p>
            <p style="margin-bottom: 1vw;">◆ <b>score:</b> Numerical score representing feature significance or intensity</p>
            <hr class="darkblue">
            <h4 class="darkblue">HiChIP loop File</h4>
            <p>◆ <b>First 6 fields:</b> Represent interacting bins for retained interactions after merge filtering.</p>
            <p>◆ <b>Next 3 fields:</b> Indicate contact count, p-value, and q-value for these interactions. </p>
            <p>◆ <b>bin1_low & bin1_high:</b> Define the lower and upper coordinates of the first set of interacting bins.</p>
            <p>◆ <b>bin2_low & bin2_high: </b> Define the lower and upper coordinates of the second set of bins.</p>
            <p>◆ <b>sumCC:</b> The total contact counts of significant interactions in the connected component model.</p>
            <p style="margin-bottom: 1vw;">◆ <b>StrongConn:</b> A value between 0 and 1, representing the fraction of significant interactions in the connected component.</p>
            <hr class="darkblue">
            <h4 class="darkblue">ChIA-PET peak file</h4>
            <p>◆ <b>chrom:</b> Chromosome name.</p>
            <p>◆ <b>summit start:</b> Start coordinate of the peak summit.</p>
            <p>◆ <b>summit end:</b> End coordinate of the peak summit.</p>
            <p>◆ <b>peak coverage:</b> Maximum tag coverage within the peak.</p>
            <p>◆ <b>p-value:</b> Statistical significance based on Poisson distribution.</p>
            <p style="margin-bottom: 1vw;">◆ <b>p.adjust:</b> Adjusted p-value using the Benjamini-Hochberg method for multiple testing.</p>
            <hr class="darkblue">
            <h4 class="darkblue">ChIA-PET loop file</h4>
            <p>◆ <b>chrom1:</b> Chromosome of cluster anchor 1.</p>
            <p>◆ <b>start1:</b> Start coordinate of cluster anchor 1.</p>
            <p>◆ <b>end1:</b> End coordinate of cluster anchor 1.</p>
            <p>◆ <b>chrom2:</b> Chromosome of cluster anchor 2.</p>
            <p>◆ <b>start2:</b> Start coordinate of cluster anchor 2.</p>
            <p>◆ <b>end2:</b> End coordinate of cluster anchor 2.</p>
            <p>◆ <b>ipet count:</b> Number of PETs linking anchors 1 and 2.</p>
            <p>◆ <b>type:</b> Interaction type (1 = intra-chromosomal, 0 = inter-chromosomal).</p>
            <p>◆ <b>distance:</b> Distance between anchors for intra-chromosomal interactions; set to 2,147,483,647 for inter-chromosomal.</p>
            <p>◆ <b>tag count within anchor 1:</b> Tags within anchor 1.</p>
            <p>◆ <b>tag count within anchor 2:</b> Tags within anchor 2.</p>
            <p>◆ <b>p-value:</b> Statistical significance of interaction (hyper-geometric distribution).</p>
            <p>◆ <b>p.adjust:</b> Adjusted p-value using Benjamini-Hochberg method.</p>
            <p>◆ <b>-log10(p-value): </b> Negative logarithm of p-value.</p>
            <p style="margin-bottom: 1vw;">◆ <b>-log10(p.adjust):</b> Negative logarithm of adjusted p-value.</p>
            <hr class="darkblue">
            <h4 class="darkblue">Annotation file</h4>
            <p>◆ <b>1-3 columns: Anchor1</b> The start and end positions of anchor1 on the chromosome, derived from processed data.</p>
            <p>◆ <b>4-6 columns: Enhancer</b> The start and end positions of the enhancer on the chromosome, obtained by intersecting anchor regions with the enhancer regions from EnhancerAtlas.</p>
            <p>◆ <b>7-9 columns: Anchor2</b> The start and end positions of anchor2 on the chromosome, derived from processed data.</p>
            <p>◆ <b>10-12 columns: Promoter</b> The start and end positions of the promoter on the chromosome, obtained by intersecting anchor regions with the promoter regions from UCSC.</p>
            <p>◆ <b>13 column: Tier1/2</b> E-P loop confidence. If the loop's anchors intersect with both the promoter region from UCSC and the enhancer region from EnhancerAtlas, it is considered Tier 1 confidence. If only one anchor intersects with the UCSC promoter region, it is considered Tier 2 confidence.</p>
            <p>◆ <b>14 column: GWAS rsID</b> The GWAS rsID for the SNP within the E-P loop.</p>
            <p>◆ <b>15 column: E/P</b> The annotation indicating whether the SNP is associated with the Promoter (P) or the Enhancer (E).</p>
            <p>◆ <b>16 column: P gene</b> The gene corresponding to the promoter of the loop.</p>
            <p>◆ <b>17 column: Rank</b> The score assigned to the SNP in RegulomeDB.</p>
            <p>◆ <b>18 column: eQTL-trait gene</b> The eQTL genes corresponding to the SNP in the eQTL Catalogue.</p>
            <p style="margin-bottom: 2vw;">◆ <b>19 column: Disease</b> The disease associated with the SNP in the GWAS Catalogue.</p>
        </div>
    </div>

    <div class="endbox" style="top: 282vw;">
        <div class="endtext">
            <div class="endtextinner">
                <p><b>XiongLab</b></p>
                <br>
                <p><b>Location:</b> Zhejiang University Medical Center, Hangzhou, China</p>
                <br>
                <p><b>Email: </b>xiongxs@zju.edu.cn</p>
                <br>
                <p><b>Contact for TM-Map: </b>yangy.21@intl.zju.edu.cn</p>
            </div>
        </div>
        <div class="endimage">
            <img style="width: 27vw;" src="./底1.png" alt="">
        </div>
    </div>
</body>
<script>
    function scrollToPosition(percentage) {
        const target = percentage * (document.documentElement.scrollHeight - window.innerHeight) / 100;
        
        if ('scrollBehavior' in document.documentElement.style) {
          window.scrollTo({
            top: target,
            behavior: 'smooth'
          });
        } else {
          const start = window.scrollY || window.pageYOffset;
          const startTime = performance.now();
          const duration = 1000; // 滚动持续时间，单位为毫秒
        
          function scroll() {
            const currentTime = performance.now();
            const timeElapsed = currentTime - startTime;
            const progress = Math.min(timeElapsed / duration, 1);
            const easing = easeOutQuart(progress);
            const newPosition = start + (target - start) * easing;
            
            window.scrollTo(0, newPosition);
        
            if (timeElapsed < duration) {
              requestAnimationFrame(scroll);
            }
          }
        
          scroll();
        }
      }
</script>
</html>